scripts/custom_lint_checks.py

```diff
@@ -297,6 +305,14 @@ def check_functiondef_params(self, node, node_doc):
             node_doc, node.args, node, node_allow_no_param)
 
     def check_functiondef_returns(self, node, node_doc):
+        """Checks whether all returns in a function definition are documented.
```

Comment:
A better way to phrase this would be:
"Checks whether a function documented with a return value actually has a return statement in its definition"

Reply:
done.